roles/docker/tasks/main.yml

```diff
@@ -85,7 +85,7 @@
   - reg_conf_var: HTTPS_PROXY
     reg_fact_val: "{{ docker_https_proxy | default('') }}"
   - reg_conf_var: NO_PROXY
-    reg_fact_val: "{{ docker_no_proxy | default('') | join(',') }}"
+    reg_fact_val: "{{ docker_no_proxy | default('') }}"
```

Comment:
I think we need to do the same in all of the templates that include openshift.common.no_proxy too if we're storing it as a string.